ark-blockstore-example/noop-plugin.go

```diff
@@ -0,0 +1,168 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"github.com/heptio/ark/pkg/cloudprovider"
+	"github.com/heptio/ark/pkg/plugin"
+	"github.com/heptio/ark/pkg/util/collections"
+	"github.com/sirupsen/logrus"
+	"math/rand"
+	"strconv"
+	// TODO: Ark plugin needs to import types from
```

Comment:
this import is fine as-is

Reply:
Since the code change is okay, I'm going to classify this as "no code change required"

Reply:
@taherv could you remove this comment? Per andy, the dependency version info is captured in dep's Gopkg.toml.

Reply:
Removed.